pkg/operator/controllers/genevalogging/genevalogging.go

```diff
@@ -26,7 +26,7 @@ const (
 	GenevaKeyName  = "gcskey.pem"
```

Comment:
Should we move these to `pkg/operator/controllers/genevalogging/const.go` to be consistent?

Reply:
Done